mr_project/mr_app/views.py

```diff
@@ -1,9 +1,231 @@
+import json
+
 from django.shortcuts import render
 
 # Create your views here.
+from time import time
+from django.http import HttpResponse, HttpResponseBadRequest, HttpResponseNotFound, JsonResponse
+import neo4j
+from django.views.decorators.csrf import csrf_exempt
+from requests import codes
+from . import DRIVER, EMOTIONS, USERS
+from datetime import datetime
+
+#WRITE TRANSACTION FUNCTIONS
+def create_user_tx(tx, name):
+    result = tx.run('CREATE (u:User{name:$name}) RETURN u.name AS username', name=name)
+    record = result.single()
+    result.consume()
+    return record['username']
+
+
+def create_emotion_tx(tx, name):
+    result = tx.run('CREATE (em:Emotion {name:$name})  RETURN em.name AS emotion_label', name=name)
+    record = result.single()
+    result.consume()
+    return record['emotion_label']
```

Comment:
There's duplicated code here that we can remove by creating a generic helper method. 
```python
def create_node_transaction(transaction, label, property, property_value):
  transaction.run(f'CREATE (node:{label} {{{property}:{property_value}}} RETURN node.{{property}}')
```